src/cai_lab_to_nwb/zaki_2024/interfaces/zaki_2024_shock_stimuli_interface.py

```diff
@@ -0,0 +1,50 @@
+"""Primary class for converting experiment-specific behavior."""
+
+from pynwb.file import NWBFile
+
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.utils import DeepDict
+from typing import Optional, List
+from pynwb.epoch import TimeIntervals
+
+
+class Zaki2024ShockStimuliInterface(BaseDataInterface):
+    """Adds annotated events of shock times."""
+
+    keywords = ["behavior", "sleep stages"]
+
+    def __init__(self, verbose: bool = False):
+
+        self.verbose = verbose
+        super().__init__()
+
+    def get_metadata(self) -> DeepDict:
+        # Automatically retrieve as much metadata as possible from the source files available
+        metadata = super().get_metadata()
+
+        return metadata
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        shock_amplitude: float,
+        shock_times: List = [120.0, 180.0, 240.0],
```

Comment:
I'd use `None` as the default for mutable arguments to avoid side effects
```
def add_to_nwbfile(..., shock_times=None):
    if shock_times is None:
        shock_times = [120.0, 180.0, 240.0]
    # Proceed with the function logic
```

Reply:
Oh, that's a good point, the other option is to use a tuple (non-mutable) and cast to list.

Reply:
Sure, that works too :)